packages/govuk-frontend/src/govuk/components/cookie-banner/cookie-banner.yaml

```
@@ -48,11 +48,11 @@ params:
           - name: type
             type: string
             required: false
-            description: The type of button. You can set `button` or `submit`. Set `button` and `href` to render a link styled as a button. If you set `href`, it overrides `submit`.
+            description: The type of button. You can set `"button"` or `"submit"`. Set `type` to `"button"` and set `href` to render a link styled as a button. If you set `href`, it overrides `type` `"submit"`.
```

This isn't quite right (it wasn't before either).

https://github.com/alphagov/govuk-frontend/blob/3cd8054a7ca9abadedae6283a4da0d0b0c73b792/packages/govuk-frontend/src/govuk/components/cookie-banner/template.njk#L42-L58

It works more like this:

```mermaid
flowchart TD
    href{"Is `href` set?"}
    href-- No -->o1["Make a button of the specified `type`"]
    href-- Yes -->hb{"Is `type` set to 'button'?"}
    hb-- No -->o2["Make a link, ignore `type` entirely"]
    hb-- Yes -->o3["Make a link that looks like a button"]
```

Maybe something like?

```suggestion
            description: The type of button – `"button"` or `"submit"`. If `href` is provided, set `type` to `"button"` render a link styled as a button.
```

---

Nice one, best of both